tools/fork-network/src/cli.rs

```diff
@@ -380,32 +402,35 @@ impl ForkNetworkCommand {
             &near_config.genesis.config,
             Some(home_dir),
         );
-        let (prev_state_roots, prev_hash, epoch_id, _block_height) =
-            self.get_state_roots_and_hash(epoch_manager.as_ref(), store.clone())?;
-        tracing::info!(?prev_state_roots, ?epoch_id, ?prev_hash);
+        let (prev_state_roots, flat_head, epoch_id, target_shard_layout) =
```

Comment:
Why the change from `prev_hash` to `flat_head`?

Reply:
So that `prev_hash` is a kind of strange naming I guess, but here we're just keeping the whole flat head `BlockInfo` rather than just the hash and height, so that we can set the flat storage status to ready w the same head for any new shards